js/models/profile/Profile.js

```diff
@@ -26,9 +26,7 @@ export default class extends BaseModel {
   }
 
   url() {
-    // url is handled by sync, but backbone bombs if I don't have
-    // something explicitly set
-    return 'use-sync';
+    return app.getServerUrl(`ob/profile/${app.profile.id}`);
```

Comment:
Shouldn't this be ${model.id}?

Currently, it looks like it gets your own profile for all profile calls.